src/components/DivElement.js

```diff
@@ -1,6 +1,17 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super();
```

Comment:
Try to take advantage of the parent constructor, and pass the variables directly into super since it already handles variable assignment.